pkg/apis/workflow/v1alpha1/pod_template_types.go

```diff
@@ -0,0 +1,65 @@
+package v1alpha1
+
+import corev1 "k8s.io/api/core/v1"
+
+type PodTemplate struct {
+	Graph        ContainerGraph       `json:"graph,omitempty" protobuf:"bytes,1,rep,name=graph"`
```

Comment:
Should this be called `DAG` to match a DAG template? Unless there's a reason that I'm missing?

Reply:
wanted to avoid name conflict with DAG, also currently allows cyclic-graph ;)

Reply:
How is possible we support cyclic graphs?